conda/activate.py

```diff
@@ -191,14 +191,8 @@ def _yield_commands(self, cmds_dict):
             yield self.run_script_tmpl % script
 
     def build_activate(self, env_name_or_prefix):
-        test_path = expand(env_name_or_prefix)
-        if isdir(test_path):
-            prefix = test_path
-            if not isdir(join(prefix, 'conda-meta')):
-                from .exceptions import EnvironmentLocationNotFound
-                raise EnvironmentLocationNotFound(prefix)
```

Comment:
What was this doing. I don;t follow

Reply:
Say you have an empty directory named `foo` in your current working directory, and also an environment named `foo` in the envs directory, that you'd normally want to use `conda activate foo` for.  This block of code would cause `conda activate foo` to fail because it would raise that `EnvironmentLocationNotFound` exception.  The logic is just incorrect here.

Reply:
Got it, thanks